### PR TITLE
Change getting started doc

### PR DESCRIPTION
**Pull request by popcor255**
# Documentation Changes
This is part of this [pull-request](https://github.com/tektoncd/pipeline/pull/2293) and [issue](https://github.com/tektoncd/website/issues/16).

### Done
* Modified contents from tutorial to readme. (I talked more about that [here](https://github.com/tektoncd/pipeline/pull/2293))
* Changed the example repo because the skaffold does not have a docker image inside the project.
* Changed task builder image from kaniko to buildah. (I think it is more user-friendly)
* Changed the image registry to docker hub instead of gcr. 
* Changed service account to user docker hub api token.
* Remove Pre-reqs or move them to before we start.

### In Progress
* Change the pipeline to support changes to the task.
* Create a Table of Contents. Cause this 'tutorial' is really like 3 tutorials in 1. (hello world w/ taskrun, build-push image w/ taskrun, build-push-deploy-image w/ pipelinerun).
* Clarify some confusion with a Task vs TaskRun and Pipeline vs PipelineRun.
* Run the entire new edited version on a clean cluster
* Change the output logs because they are out of date

### Comments
* There is a 'surprise' task under the pipeline section that will allow the user to deploy a simple image. I think cramming this task at the end of the pipeline section without a lot of explanation is confusing. Especially, since the word 'deployment' was not even mentioned until the very end. So, if anyone has sneaky ideas of introducing this task please let me know.

TL;DR: I once told a man, "I will get it done by lunch". ***I lied***

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -151,51 +141,71 @@ the `Task` definition is constant and the value of parameters can change during
...
+    - name: varlibcontainers
+      emptyDir: {}
 ```
 
 ### Configuring `Task` execution credentials
 
 Before you can execute your `TaskRun`, you must create a `secret` to push your image
 to your desired image registry:
 
+*** Note: *** You can get your docker access token at [docker hub](https://hub.docker.com/settings/security).
```
Comment (sergetron) [inline docs/getting-started.md]: This doesn't render properly. The correct syntax for bolding is two asterisks on each side --> **Note:**

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,27 +1,17 @@
...
 
-This tutorial uses a simple `Hello World` example to show you how to:
-- Create a `Task`
-- Create a `Pipeline` containing your `Tasks`
-- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`
-- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+This guide will get you started with tekton pipelines with a simple `Hello World` tutorial.
 
-This tutorial consists of the following sections:
+## Before we begin
```
Comment (sergetron) [inline docs/getting-started.md]: "Before you begin." It's the reader doing the tutorial, not us.

File: docs/getting-started.md [hunk 3]
```diff
@@ -1,27 +1,17 @@
-# Tekton Pipelines Tutorial
+# Getting Started with Tekton Pipelines
 
-This tutorial uses a simple `Hello World` example to show you how to:
-- Create a `Task`
-- Create a `Pipeline` containing your `Tasks`
-- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`
-- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+This guide will get you started with tekton pipelines with a simple `Hello World` tutorial.
```
Comment (sergetron) [inline docs/getting-started.md]: The bullet list above gives a clear overview of what the reader will accomplish in the tutorial. I think you should restore it, because the new sentence isn't very informative. You also need to capitalize "Tekton Pipelines".

File: docs/getting-started.md [hunk 4]
```diff
@@ -1,27 +1,17 @@
...
 
-This tutorial consists of the following sections:
+## Before we begin
 
-- [Creating and running a `Task`](#creating-and-running-a-task)
-- [Creating and running a `Pipeline`](#creating-and-running-a-pipeline)
+Before we begin this tutorial, make sure you have [installed and configured](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)
```
Comment (sergetron) [inline docs/getting-started.md]: "You" instead of "we". It's the reader completing the steps, not someone else together with them.

File: docs/getting-started.md [hunk 5]
```diff
@@ -1,27 +1,17 @@
...
 
-**Note:** Items requiring configuration are marked with the `#configure` annotation.
-This includes Docker registries, log output locations, and other configuration items
-specific to a given cloud computing service.
+If you would like to complete this tutorial on your local workstation, make sure you run clean profile of [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) v1.50 or higher.
 
-## Before you begin
+You will also need an account on [docker hub](https://hub.docker.com).
 
-Before you begin this tutorial, make sure you have [installed and configured](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)
```
Comment (sergetron) [inline docs/getting-started.md]: What was wrong with this section? I think you're removing useful information here.

File: docs/getting-started.md [hunk 6]
```diff
@@ -1,27 +1,17 @@
...
-the latest release of Tekton on your Kubernetes cluster, including the
-[Tekton CLI](https://github.com/tektoncd/cli).
-
-If you would like to complete this tutorial on your local workstation, see [Running this tutorial locally](#running-this-tutorial-locally). To learn more about the Tekton entities involved in this tutorial, see [Further reading](#further-reading).
+Also, make sure you are in the tekton-pipelines namespace if you have tekton installed.
```
Comment (sergetron) [inline docs/getting-started.md]: Don't use "also" repeatedly. A bulleted list is much easier to parse than "Also this," "also that."

File: docs/getting-started.md [hunk 7]
```diff
@@ -1,27 +1,17 @@
...
+Before we begin this tutorial, make sure you have [installed and configured](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)
+the latest release of Tekton on your Kubernetes cluster, including the [Tekton CLI](https://github.com/tektoncd/cli). 
 
-**Note:** Items requiring configuration are marked with the `#configure` annotation.
-This includes Docker registries, log output locations, and other configuration items
-specific to a given cloud computing service.
+If you would like to complete this tutorial on your local workstation, make sure you run clean profile of [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) v1.50 or higher.
 
-## Before you begin
+You will also need an account on [docker hub](https://hub.docker.com).
```
Comment (sergetron) [inline docs/getting-started.md]: Capitalize "Docker".

File: docs/getting-started.md (hunk 5 above)
Comment (popcor255) [reply docs/getting-started.md]: I moved it. I did not completely remove it.

Comment (sergetron) [reply docs/getting-started.md]: Sorry, you're right, my mistake.

File: docs/getting-started.md (hunk 3 above)
Comment (popcor255) [reply docs/getting-started.md]: fixed on new commit

Comment (popcor255) [reply docs/getting-started.md]: I will resolve this if you are ok with the change I made.

**Comment by sergetron** [comment]
Thank you for all the updates! Could you squash all the commits before we LGTM?

**Comment by popcor255** [comment]
Yeah, I need to rebase with master, squash commits and run this on local a few times before changing it from draft to a full pr.

**Comment by popcor255** [comment]
oof, I goofed up the commit history.
